src/annotation_operators.ts

```diff
@@ -11,13 +17,264 @@ export function get_annotation_confidence(annotation) {
     return current_confidence;
 }
 
-//Takes in an annotation and marks it either deprecated or not deprecated.
-export function mark_deprecated(annotation: ULabelAnnotation, deprecated: boolean) {
+/**
+ * Takes in an annotation and marks it either deprecated or not deprecated.
+ * 
+ * @param annotation ULabelAnnotation
+ * @param deprecated boolean 
+ */
+export function mark_deprecated(annotation: any, deprecated: boolean) {
     annotation.deprecated = deprecated
 }
 
-//if the annotation confidence is less than the filter value then return true, else return false
-export function filter_low(annotation_confidence: number, filter_value: number) {
-    if (annotation_confidence < filter_value) return true
+/**
+ * If the value is less than the filter then return true, else return false.
+ * 
+ * @param value Value to be compaired against the filter
+ * @param filter What the value is compared against
+ */
+export function filter_low(value: number, filter: number) {
+    if (value < filter) return true
     return false
+}
+
+/**
+ * If the value is greater than the filter then return true, else return false.
+ * 
+ * @param value Value to be compaired against the filter
+ * @param filter What the value is compared against
+ */
+export function filter_high(value: number, filter: number) {
+    if (value > filter) return true
+    return false
+}
```

Comment:
The names `filter_low`/`filter_high` imply to me that values are actually being filtered, whereas this is more of a check to see if a `value` is lower/higher than the `filter`. Maybe use names like `value_is_lower_than_filter` or smth?